doc/README.md

```diff
@@ -189,8 +192,46 @@ paths with the highest cum weight.
 * **-svg:** Generates a report in SVG format.
 * **-web:** Generates a report in SVG format on a temp file, and starts a web
   browser to view it.
-* **-png, -jpg, -gif, -pdf:** Generates a report in these formats,
+* **-png, -jpg, -gif, -pdf:** Generates a report in these formats.
+
+### Interpreting the Callgraph
+
+* **_flat_ value**: the value of a location itself, is indicated by the box size.
+
+    *   Nodes with larger box have larger _flat_ values.
```

Comment:
Typically only the first occurrence of a new term is rendered in italic font in texts. Subsequent occurrences do not need to be in italic.

Reply:
And the terms are introduced above at the beginning of the "Details" section.

Reply:
Done with changing the italic font.

Maggies also commented before on the re-definition of those terms and I think I agree with her that a bit redundancy is okay, especially since this section will also be referred to from other places.